http/grpc/src/main/java/io/quarkus/ts/http/grpc/GrpcReflectionResponse.java

```diff
@@ -1,7 +1,10 @@
-package io.quarkus.ts.http.advanced.reactive;
+package io.quarkus.ts.http.grpc;
 
 import java.util.List;
 
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection
```

Comment:
This class is also return type for JAX-RS resource endpoint, why is registering for reflection necessary? Is it possibly a bug?

Reply:
It was failing before, but now works fine. Weird, but I probably missed something. Changed